src/FakeItEasy/Configuration/PropertySetterConfiguration.cs

```diff
@@ -101,10 +78,8 @@ public IPropertySetterConfiguration WhenArgumentsMatch(Func<ArgumentCollection,
         }
 
         private static IPropertySetterConfiguration AsPropertySetterConfiguration(
-            IVoidConfiguration voidArgumentValidationConfiguration)
-        {
-            return new ProperySetterAdapter(voidArgumentValidationConfiguration);
-        }
+            IVoidConfiguration voidArgumentValidationConfiguration) =>
+            new ProperySetterAdapter(voidArgumentValidationConfiguration);

```

Comment:
Actually, that is the way to do it. The alternative I suggested doesn't convey the nesting, but this does. And Visual Studio reformatting is happy with it. I'll push another fixup.